homeassistant/components/supla/__init__.py

```diff
@@ -2,17 +2,23 @@
 import logging
 from typing import Optional
 
-from pysupla import SuplaAPI
+import async_timeout
+from asyncpysupla import SuplaAPI
 import voluptuous as vol
 
-from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.const import (
+    CONF_ACCESS_TOKEN,
+    CONF_SCAN_INTERVAL,
+    EVENT_HOMEASSISTANT_CLOSE,
```

Comment:
```suggestion
    EVENT_HOMEASSISTANT_STOP,
```

Reply:
Thanks for suggestions, I'm committing updates now.